scripts/fetch-package.js

```diff
@@ -15,7 +15,7 @@ const PUB_KEY_URL = "https://packages.riot.im/element-release-key.asc";
 const PACKAGE_URL_PREFIX = "https://github.com/vector-im/element-web/releases/download/";
 const ASAR_PATH = 'webapp.asar';
 
-const {setPackageVersion, setDebVersion} = require('./set-version.js');
+const setPackageVersion = require('./set-version.js');
```

Comment:
Looking at how [`setPackageVersion` is exposed in `./set-version.js`](https://github.com/vector-im/element-desktop/blob/develop/scripts/set-version.js#L55)

It appears that we would need destructuring here

```suggestion
const {setPackageVersion} = require('./set-version.js');
```